sdk/rpc/server/server.go

```diff
@@ -116,16 +117,25 @@ func (t *RPCServer) ExecutableNeedsAuth(req proto.ExecutableNeedsAuthRequest, re
 
 // CredentialImport is a remote version of the Import() function in schema.CredentialType.
 // The call is forwarded to the Import() function of the credential identified by req.CredentialID.
-func (t *RPCServer) CredentialImport(req proto.ImportCredentialRequest, resp *sdk.ImportOutput) error {
+func (t *RPCServer) CredentialImport(req proto.ImportCredentialRequest, resp proto.ImportCredentialResponse) error {
```

Comment:
The `net/rpc` package [defines](https://pkg.go.dev/net/rpc) that the second argument (`resp`) has to be a pointer. The reason is that the second argument is used as a return value. If it's not a pointer, any changes to it will be lost after this function returns.

Reply:
Done!